pkg/constants/constants.go

```diff
@@ -61,4 +61,7 @@ const (
 
 	JVMUserConfigMapName = "jvm-build-config"
 	JVMEnableRebuilds    = "enable-rebuilds"
+
+	QUAY_OAUTH_TOKEN_RELEASE_SOURCE      string = "QUAY_OAUTH_TOKEN_RELEASE_SOURCE"
```

Comment:
This is not a proper name for variables in go. They should be camel case, no underscore and start with capital letter or not depending on if you want to export them or not.